pingdom-checks/data-query.ping

```diff
@@ -37,3 +37,19 @@ pingdom save-check \
   -a url="/services/fhir/v0/dstu2/Patient/1011537977V693883" \
   -a authorization_token="$HEALTH_APIS_STATIC_ACCESS_TOKEN" \
   -a integrationids_csv="$HEALTH_APIS_SLACK_ID";
+
+  pingdom save-check \
+    --template simple-resource \
+    -a name=sandbox-health-alb \
+    -a host=sandbox-api.va.gov \
+    -a url="/services/fhir/v0/healthz" \
+    -a group="health-apis" \
+    -a integrationids_csv="$HEALTH_APIS_SLACK_ID";
+
+  pingdom save-check \
+    --template fhir-resource \
+    -a name=sandbox-dstu2-patient \
```

Comment:
Do we want to add `Condition` and `Med-Order` checks here like we have for production?

Reply:
we don't have them today, so I'll say not at this time.
there has been a discussion with Gabe on what the right level to monitor resources is, we may end up doing all of them, but for now, let's just mimic what exists today